external/imgui.cmake

```diff
@@ -4,11 +4,13 @@ set_property (DIRECTORY PROPERTY EP_BASE Dependencies)
 
 set (DEPENDENCIES)
 #set (IMGUI_VERSION 6a0d0dab5a9f0b9518a2bc9bb456a69895ae0962) # tag: v1.72b, 2019/07/31
-set (IMGUI_VERSION tables)
+#set (IMGUI_VERSION tables)
+set (IMGUI_VERSION main)
 message (STATUS "Adding imgui ${IMGUI_VERSION} as an external project.")
 
 ExternalProject_Add(imgui
-  GIT_REPOSITORY https://github.com/ocornut/imgui.git
+  #GIT_REPOSITORY https://github.com/ocornut/imgui.git
+  GIT_REPOSITORY https://github.com/ahmedskhalil/imgui_e3ed9ef1b.git
```

Comment:
We would want to use the ocornut repo.  If we need to set a particular version, we should do that instead of hosting a branch ourselves.

Also, does main now incorporate the tables features?

Reply:
The required commit `e3ed9ef1bf115532762e90a621d9a3b5c5b39e25` which the `develop` was based on is no longer fetchable, when the `superbuild` checks out this commit it doesn't find it. The `tables`-> [New Tables API (alpha available for testing) #2957](https://github.com/ocornut/imgui/issues/2957) branch is constantly updating and still in the experimental phase, because of that the CI was failing as the API for creating tables has completely changed (`Widget.cpp:L125-L190`) -> [CI](https://app.circleci.com/pipelines/github/dmccloskey/SmartPeak2/1241/workflows/c7d1e1c2-9713-464c-8111-fbd7635724cf/jobs/1642/parallel-runs/0/steps/0-108) . I have temporarily changed the ImGui to point to this exact version until we update the `Widget.cpp` and any other files affected by this change as a hotfix. Would you suggest integrating the latest ImGui `tables` branch into this PR and later into our `develop` branch? It'll have the advantage of using ImGui table sorting capabilities.

Reply:
Let's do the following: 1) integrate the latest ImGui `tables` branch into a different PR and then merge into develop (i.e., https://biosustain-dev.atlassian.net/browse/AUT-586), 2) add in the table sorting capabilities in another PR and then merge into develop (i.e., as a part of https://biosustain-dev.atlassian.net/browse/AUT-424).

Reply:
FYI I've been rebasing+force pushing `tables` a lot during dev. Releasing 1.79 today and expecting to merge tables into master in 1.80 maybe next month.